CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 Here we document changes that affect the public API or changes that needs to be communicated to other developers. 
 
+## 2022-03-09 DataFrame filtering and CSV filtering
+DataFrames can now be filtered with the `DataFrame Filter` processor or when loading a CSV file with the `CSV Source` processor. The `DataFrame Filter` also supports brushing & linking filtering. Filters exist for matching values (`int64` and `double` as well as categorical values), ranges, and commented lines.
+
```

Comment:
Could add breaking changes and how to resolve them here.

Reply:
Will add the changes in `CategoricalColumn`.